### PR TITLE
[com_installer] Add search tooltip to all views

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Summary of Changes

Simple PR to improve the search tooltip and element label (accessibility) in com_installer.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Extensions -> Manage -> Update, check the tooltip in the search field.
3. Go to Extensions -> Manage -> Manage, check the tooltip in the search field.
4. Go to Extensions -> Manage -> Discover, check the tooltip in the search field.
5. Go to Extensions -> Manage -> Install Languages, check the tooltip in the search field.
6. Go to Extensions -> Manage -> Update Sites, check the tooltip in the search field.

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_installer.ini [hunk 1]
```diff
@@ -10,6 +10,8 @@ COM_INSTALLER_CACHETIMEOUT_LABEL="Updates Caching (in hours)"
 COM_INSTALLER_CONFIGURATION="Installer: Options"
 COM_INSTALLER_CONFIRM_UNINSTALL="Are you sure you want to uninstall? Confirming will permanently delete the selected item(s)!"
 COM_INSTALLER_CURRENT_VERSION="Installed"
+COM_INSTALLER_DISCOVER_FILTER_SEARCH_DESC="Search in discovered extension name. Prefix with ID: to search for a extension ID."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_installer.ini]: please change to 
an extension

File: administrator/language/en-GB/en-GB.com_installer.ini [hunk 2]
```diff
@@ -74,6 +76,9 @@ COM_INSTALLER_LABEL_HIDEPROTECTED_DESC="Hide protected extensions. Protected ext
 COM_INSTALLER_LABEL_HIDEPROTECTED_LABEL="Hide Protected Extensions"
 COM_INSTALLER_LANGUAGES_AVAILABLE_LANGUAGES="Available Languages"
 COM_INSTALLER_LANGUAGES_FILTER_SEARCH_DESC="Search by language name."
+COM_INSTALLER_LANGUAGES_FILTER_SEARCH_LABEL="Search Languages"
+COM_INSTALLER_MANAGE_FILTER_SEARCH_DESC="Search in extension name. Prefix with ID: to search for a extension ID."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_installer.ini]: please change to 
an extension

File: administrator/language/en-GB/en-GB.com_installer.ini [hunk 3]
```diff
@@ -225,10 +230,14 @@ COM_INSTALLER_UNABLE_TO_FIND_INSTALL_PACKAGE="Unable to find install package"
 COM_INSTALLER_UNINSTALL_ERROR="Error uninstalling %s."
 COM_INSTALLER_UNINSTALL_LANGUAGE="A language should always have been installed as a package. <br />To uninstall a language, filter type by package and uninstall the package."
 COM_INSTALLER_UNINSTALL_SUCCESS="Uninstalling the %s was successful."
+COM_INSTALLER_UPDATE_FILTER_SEARCH_DESC="Search in extension name. Prefix with ID:, UID: or EID: to search for a update ID, update site ID: or extension ID."
+COM_INSTALLER_UPDATE_FILTER_SEARCH_LABEL="Search Extensions with Updates"
 COM_INSTALLER_UPDATESITE_DISABLE="Disable update site"
 COM_INSTALLER_UPDATESITE_DISABLED="Disabled update site"
 COM_INSTALLER_UPDATESITE_ENABLE="Enable update site"
 COM_INSTALLER_UPDATESITE_ENABLED="Enabled update site"
+COM_INSTALLER_UPDATESITES_FILTER_SEARCH_DESC="Search in extension name. Prefix with ID: to search for a update site ID."
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_installer.ini]: please change to 
an update

**Comment by brianteeman** [comment]
Added some inline comments about en-gb strings

**Comment by andrepereiradasilva** [comment]
done

**Comment by brianteeman** [comment]
I have tested this item :red_circle: unsuccessfully on 30ac6079e51d16735645c88d1d4ba8c2777ce4ff<br /><br />Manage
search by ID doesnt work

Discover
search by ID doesnt work

Update sites
search by ID doesnt work
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10174">issues.joomla.org/joomla-cms/10174</a>.</sub>

**Comment by andrepereiradasilva** [comment]
test now please. It was not working (bug) before this PR.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e5ade7f6c45c18a9b3ff5dee15fffc4bc0fbb06d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10174">issues.joomla.org/joomla-cms/10174</a>.</sub>

File: administrator/language/en-GB/en-GB.com_installer.ini [hunk 4]
```diff
@@ -225,10 +230,14 @@ COM_INSTALLER_UNABLE_TO_FIND_INSTALL_PACKAGE="Unable to find install package"
 COM_INSTALLER_UNINSTALL_ERROR="Error uninstalling %s."
 COM_INSTALLER_UNINSTALL_LANGUAGE="A language should always have been installed as a package. <br />To uninstall a language, filter type by package and uninstall the package."
 COM_INSTALLER_UNINSTALL_SUCCESS="Uninstalling the %s was successful."
+COM_INSTALLER_UPDATE_FILTER_SEARCH_DESC="Search in extension name. Prefix with ID:, UID: or EID: to search for a update ID, update site ID: or extension ID."
```
Comment (grhcj) [inline administrator/language/en-GB/en-GB.com_installer.ini]: Last ":" should be "," or has to be deleted.

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_installer.ini]: No it is correct

The string you have to insert is XX:

Comment (brianteeman) [inline administrator/language/en-GB/en-GB.com_installer.ini]: Sorry missed the part you were referring to. The last : should be a , 

Good spot

**Comment by andrepereiradasilva** [comment]
nice spoted @grhcj !  fixed

**Comment by grhcj** [comment]
I have tested this item :white_check_mark: successfully on e161b6d0c6186c5a654f2c2e0beed586ad902efc<br /><br />Completed testing on code review (only string changes)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10174">issues.joomla.org/joomla-cms/10174</a>.</sub>

**Comment by andrepereiradasilva** [comment]
@brianteeman RTC? the only think changed since your test is the removal of the extra ":". https://github.com/joomla/joomla-cms/pull/10174/commits/e161b6d0c6186c5a654f2c2e0beed586ad902efc

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on e161b6d0c6186c5a654f2c2e0beed586ad902efc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10174">issues.joomla.org/joomla-cms/10174</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10174">issues.joomla.org/joomla-cms/10174</a>.</sub>

**Pull request merged**